.github/PULL_REQUEST_TEMPLATE/analyst_experience_team.md

```diff
@@ -0,0 +1,48 @@
+## Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+- **Related issue:** < LINK_TO_RELATED_ISSUE >
+- **Feature flag:** < FEATURE_FLAG_HERE >
+- **Docs ticket:** < LINK_TO_DOCS_TICKET_HERE >
+- **Schema changes:** < LINK_SCHEMA_CHANGE_TICKET_HERE >
+
+### How to test
+
+Provide steps for reviewers to know how to test your PR. Assume that a reviewer may have little to no context when determining how detailed this section should be.
+
+### Release Notes
+
+The text that appears in the Release Notes is pulled directly from your PR title, or a single paragraph of text that you specify in the PR description. See [Release Notes Guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process).
+
+### Definition of done
+
+Check the PR satisfies following conditions.
+
+- [ ] Functional changes are hidden behind a feature flag. If not hidden, the PR explains why these changes are being implemented in a long-living feature branch. - [Feature flag guidelines](https://docs.elastic.dev/security-solution/process/detections/pull-requests#feature-toggling-for-everything)
+- [ ] Functional changes are covered with a test plan and automated tests.
```

Comment:
For the automated tests, would it be possible to link to our [best practices documentation](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/best-practices/README) or add a new point or clarification to ensure that people follow it?